android/app/src/main/java/app/organicmaps/DownloadResourcesLegacyActivity.java

```diff
@@ -231,7 +232,7 @@ protected void onSafeDestroy()
     super.onSafeDestroy();
     mApiRequest.unregister();
     mApiRequest = null;
-    Utils.keepScreenOn(false, getWindow());
+    Utils.keepScreenOn(Config.isKeepScreenOnEnabled(), getWindow());
```

Comment:
This is a good question. I think that the intention was to keep screen on because the downloader doesn't work in the background. I also thought about removing this hack...

Reply:
It looks like that this hack is still needed, because otherwise the screen goes off during DownloadResourcesLegacyActivity (the world.mwm downloader) and aborts downloading... Yes, a better solution is needed, but let's address it separately. This PR doesn't change the behavior there.

Reply:
There are many ways to keep a specific window/view on without wakelocks, why they can't be applied to the world mwm downloader? https://stackoverflow.com/a/18487237

Reply:
Downloader should use a foreground service. But it is a different issue.